lib/runner/extract-runnable-items.js

```diff
@@ -186,6 +186,11 @@ var sdk = require('postman-collection'),
 
         items = findItemsOrGroups(collection, entrypointLookup, true);
 
+        // Sorting the items by the order in which they appear as folder arguments.
+        items && items.members && items.members.sort(function (a, b) {
```

Comment:
The mere existence of the `members` property does not guaranty the existence of the `sort` function. `members` must be an array.